src/test/datascience/kernelLauncher.vscode.test.ts

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+'use strict';
+
+import { assert, use } from 'chai';
```

Comment:
Moved to VS Code tests to use the real ioc, tests are faster here
The tests were pretty slow in functional tests as for every single test we were starting the IOC and a lot of code was running in the background to load styles, etc... (lots in the logs for every test).

This seem to have fixed the test as well - i just moved it to run as a VS Code test & works.